app/webpacker/css/admin/products_v3.scss

```diff
@@ -21,13 +21,37 @@
 
   // Form actions floats over other controls when active
   .form-actions {
-    position: absolute;
-    top: -1em;
+    position: sticky;
+    top: 0;
     left: 0;
     right: 0;
     z-index: 1; // Ensure tom-select and .disabled-section are covered
   }
 
+  .form-actions:not(.hidden) + table.products {
+    thead {
+      top: 54px;
+
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 50%;
+        right: 50%;
+        width: 102%;
+        height: 0.5rem;
+        background-color: white;
+        transform: translate(-50%, -100%);
+      }
+    }
+
+    @media screen and (max-width: 984px) {
+      thead {
+        top: 86px;
+      }
```

Comment:
I can see that this fix is because the "Save changes" button wraps to a new line on small screen widths. It doesn't need to though, so I think we should fix that instead.

I have an idea to fix it, will add a new commit to demonstrate.